### DISCUSSION TITLE
Modify MINIO_VOLUMES ??

### DISCUSSION THREAD

**Discussion by c3b0la** [minio/minio#13379] [Q&A] [answered]
Hello, How should I change the MINIO_VOLUMES repository after the MiniO is installed?

I installed Minio, and used a repository from the Tutorial example and now I can't change to the repository I had reserved. I tried to edit the file /etc/default/minio but I still have a problem, when I restart the service, the MinIO does not start.

I need help and forgive me if I'm posting my question in the wrong way, just new in the community.
Thanks

**Comment by harshavardhana** [comment, marked as answer]
You need to wipe your disks if you want to change your installation @c3b0la

**Reply by c3b0la** [reply]
@harshavardhana Thanks for the answer, when you say wipe the disk, you say remove MinIO or format the server?

**Reply by harshavardhana** [reply]
Stop MinIO, wipe the drives - the error message would tell you why it cannot start. Remember you can't just change MINIO_VOLUMES  unless you are doing a pool expansion.

**Reply by c3b0la** [reply]
@harshavardhana 
Ok, I'll follow your recommendation, I'll get back to you soon with feedback!
Thank you!
